corehq/motech/fhir/docs/index.rst

```diff
@@ -0,0 +1,185 @@
+MOTECH on FHIR
+==============
+
+CommCare HQ offers two ways of sharing data over FHIR:
+#. The FHIR API exposes CommCare cases as FHIR resources.
+#. Data forwarding allows CommCare cases to be sent to remote FHIR
+   services.
+
+FHIR-related functionality is currently enabled using the "FHIR
+integration" feature flag.
+
+
+The FHIR API
+------------
+
+MOTECH offers a FHIR R4 API. It returns responses in JSON.
+
+The API focuses on the Patient resource. The endpoint for a Patient with
+case ID 11111111 would be
+https://www.commcarehq.org/a/PROJECT-SPACE/fhir/R4/Patient/11111111
+
+(Throughout this documentation, we will use "PROJECT-SPACE" as a
+placeholder value for the name of a project space on CommCare HQ.)
+
+To search for the patient's Observations, the API accepts the
+"patient_id" search filter. For example,
+https://www.commcarehq.org/a/PROJECT-SPACE/fhir/R4/Observation/?patient_id=11111111
+
+The FHIR API CapabilityStatement can be found at
+https://www.commcarehq.org/a/PROJECT-SPACE/fhir/R4/metadata
```

Comment:
I think you need to implement this. I removed the SMART-on-FHIR specific metadata endpoint.